src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -81,6 +92,12 @@ export const FilterSidebar = ({
       easing: easings.easeOutSine,
     },
   })
+
+  const hideSearch = useMemo(
+    () => walletCollections && walletCollections?.length >= WALLET_COLLECTIONS_PAGINATION_LIMIT,
+    [walletCollections]
```

Comment:
this won't take effect unless a user scroll so that they have > 300 collections loaded, I'd suggest passing in `hasNextPage` from the infinite query and then checking if that's true and they have 300 collections loaded

Reply:
it will, because its greater than or EQUAL to. I tested with pranksy's wallet in simulator. also hasNextPage will become false at some point at the bottom of the scroll, which leads to the search bar appearing again (undesirable effect, can confuse users)